### PR TITLE
*: add get label information

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @siddontang @andelf @huachaohuang 
close https://github.com/pingcap/pd/issues/505 https://github.com/pingcap/pd/issues/507

### CODE REVIEW

File: pdctl/command/global.go [hunk 1]
```diff
@@ -26,7 +27,56 @@ import (
...
+}
+
+func validPDAddr(pd string) error {
+	u, err := url.Parse(pd)
+	if err != nil {
+		return err
+	}
+	if u.Scheme == "" {
+		u.Scheme = "http"
+	} else if u.Scheme != "http" {
```
Comment (siddontang) [inline pdctl/command/global.go]: we will support https later

File: pdctl/command/label_command.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+var (
+	labelsPrefix      = "pd/api/v1/labels"
+	labelsStorePrefix = "pd/api/v1/labels/store"
+)
+
+// NewLabelCommand return a member subcommand of rootCmd
+func NewLabelCommand() *cobra.Command {
+	l := &cobra.Command{
+		Use:   "label [store <name> <value>]",
```
Comment (siddontang) [inline pdctl/command/label_command.go]: I think value is optional.

**Comment by siddontang** [comment]
@nolouch 
Can you split the PR into two, one is for simplifying HTTP handling, the other is for label.

**Comment by nolouch** [comment]
I will split this PR

**Pull request closed without merging**